src/scripts/public/single.js

```diff
@@ -0,0 +1,41 @@
+import { qs, addSafeEventListener } from './utils';
+
+/**
+ * Single post manipulation.
+ * [@return](https://github.com/return) {void}
+ */
+const bcgovBlockThemeSinglePost = () => {
```

Comment:
This does seem like a lot of computation to do a default layout for a post. Surely there's an elegant way to do it using only CSS, (assuming the existing CSS isn't so complicated and messy that it's impossible to make this change without a major refactor).

Reply:
Special request from the King himself. Nate King that is. Dvh is used in several places within this repo. I'm not opposed to researching and reviewing the usage, but I think that would be a separate ticket.

Reply:
CSS is getting better and better but it still lacks in some ways. Occasionally you have to dip into JS to calculate things you won't be able to surface through CSS alone.